app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -279,7 +279,7 @@ def self.raw_alerts_connect(hostname, port, username, password)
         :username => username,
         :password => password
       }
-      ::Hawkular::Alerts::AlertsClient.new(url, credentials)
+      ::Hawkular::Alerts::Client.new(url, credentials)
```

Comment:
I also grep'ed managui-ui-classic repo and found no references to it.

Anyway... The code using `raw_alerts_connect` will fail with an exception because we have this: https://github.com/hawkular/hawkular-client-ruby/blob/master/lib/hawkular/base_client.rb#L32

I think it's dead code and I can safely remove it.